### ISSUE TITLE
MultipartFormData is not present in json export

### ISSUE THREAD

**Issue by jcamiel** [Orange-OpenSource/hurl#63]
MultipartFormData is not present in json export.

With this file:

```
POST http://localhost:8000/multipart-form-data
[MultipartFormData]
key1: value1
upload1: file,hello.txt;
upload2: file,hello.html;
upload3: file,hello.txt; text/html

HTTP/1.0 200
```

and

```
hurl --json /tmp/test.json -v integration/tests/multipart_form_data.hurl
```

The report is :

```json
[
  {
    "filename": "integration/tests/multipart_form_data.hurl",
    "entries": [
      {
        "request": {
          "method": "POST",
          "url": "http://localhost:8000/multipart-form-data",
          "queryString": [],
          "headers": [],
          "cookies": [],
          "body": ""
        },
        "response": {
          "httpVersion": "HTTP/1.0",
          "status": 200,
          "cookies": [],
          "headers": [
            {
              "name": "Content-Type",
              "value": "text/html; charset=utf-8"
            },
            {
              "name": "Content-Length",
              "value": "0"
            },
            {
              "name": "Server",
              "value": "Werkzeug/1.0.1 Python/3.8.5"
            },
            {
              "name": "Date",
              "value": "Mon, 02 Nov 2020 13:00:52 GMT"
            }
          ]
        },
        "captures": [],
        "asserts": [
          {
            "actual": "1.0",
            "expected": "1.0"
          },
          {}
        ],
        "time": 9
      }
    ],
    "success": true,
    "time": 9,
    "cookies": []
  }
]
```

**Comment by fabricereix** [comment]
The HAR structure for multipart form data params can be used.
The example above will be expressed by:

```
"multipart_form_data": [
    {
        "name": "key1",
        "value": "value1"
    },
    {
        "name": "upload1",
        "value": "Hello World!",
        "fileName": "hello.txt",
        "contentType": "text/plain"
    },
    {
        "name": "upload2",
        "value": "Hello <b>World</b>!",
        "fileName": "hello.html",
        "contentType": "text/html"
    },
    {
        "name": "upload3",
        "value": "Hello <b>World</b>!",
        "fileName": "Hello World!",
        "contentType": "text/html"
    }
]
```

**Comment by fabricereix** [comment]
Note that the value is optional (same than HAR)
for example, the data from the file might not be a UTF8 encoded String. In this case, the value field will not be included.

**Comment by jcamiel** [comment]
Looks good! Small point: if possible, we should not mix snake case (`multipart_form_data`) and camel case (`fileName`) (unless it's the case in the HAR format)

**Comment by fabricereix** [comment]
HAR is definitely camel case, and I tend to prefer snake case for json
but yes we should not mix and stick to camel case

**Comment by fabricereix** [comment]
fixed in #67
